pkg/session/bootstrap_test.go

```diff
@@ -270,6 +270,13 @@ func revertVersionAndVariables(t *testing.T, se sessiontypes.Session, ver int) {
 		// for version <= version195, tidb_enable_dist_task should be disabled before upgrade
 		MustExec(t, se, "update mysql.global_variables set variable_value='off' where variable_name='tidb_enable_dist_task'")
 	}
+	if ver < version212 {
```

Comment:
only for tests?  if in real environment, do we have a way or need a way to revert?

Reply:
Yes, for testing purposes only. We will not encounter this condition in the real world environment. PTAL, thx~ @tangenta